packages/terra-clinical-label-value-view/lib/LabelValueView.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
+
+.terraClinical-LabelValueView {
+  border: 0;
+  padding: 0.1785rem;
+}
+
+.terraClinical-LabelValueView-label {
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 20px;
+}
+
+.terraClinical-LabelValueView-value {
+  font-size: 1.4285714285714286rem;
+  line-height: 1.4285714285714286rem;
```

Comment:
Dave brought up a good point about this styling. If a String is passed in as a value we should style it based on UX feedback, but if a component was passed in, we probably shouldn't apply a font-size and line-height.